docs/intercommunication.rst

```diff
@@ -0,0 +1,30 @@
+
+Intercommunication details
+==========================
+Details about integration points that will be routed through the commerce coordinator
+
+Order History Page
+~~~~~~~~~~~~~~~~~~
+We link to an order history page from a user dropdown menu on non-staff accounts (frontend-component-header’s AuthenticatedUserDropdown
+component), and the destination url is: https://orders.edx.org/orders
+
+This order history page is served from the frontend-app-ecommerce repo. Here are the steps to retrieve that data:
+
+1. When the OrderHistoryPage component mounts, it calls fetchOrders.
+
+2. That fetchOrders action is set defined in `actions.js`_. The `saga`_ uses generator functions to call getOrders and yield objects to the middleware,
```

Comment:
Isn't redux-saga the middleware? `handleFetchOrders` dispatches the action with the result that it gets from the API call?